docs/codeql/codeql-workshops/.gitignore

```diff
@@ -0,0 +1,6 @@
+# macOS indexing
```

Comment:
I think these are covered by the root `.gitignore`.

Reply:
rm'd